_posts/2022-07-18-kubeflow-1.5.1-release.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Kubeflow 1.5.1 Release Announcement"
+layout: post
+toc: true
+comments: true
+image: images/logo.png
+hide: false
+categories: [release]
+permalink: /kubeflow-1.5-release/
+author: "Kubeflow 1.5 Release Team"
+---
+
+The Kubeflow Community is excited to announce the release of [Kubeflow 1.5.1](https://github.com/kubeflow/manifests/releases/tag/v1.5.1). The Kubeflow 1.5.1 release provides a valuable enhancement to the caching feature of Kubeflow Pipeline (KFP) for any Kubernetes distribution with strict security policies for user application certificates. Caching is one of the crucial features of KFP which enables skipping a step that has already been executed which saves time and money. This release enables the caching feature to be used on distributions like AWS, Mercedes Benz cluster etc.
```

Comment:
> This release enables the caching feature to be used on distributions like AWS, Mercedes Benz cluster etc.

Can we rephrase it by replacing `enables` with `unblocks`?

Reply:
Done!